lib/scnnr/client.rb

```diff
@@ -18,6 +18,7 @@ def config
     def recognize_image(image, options = {})
       PollingManager.start(self, merge_options(options)) do |opts|
         uri = construct_uri('recognitions', opts)
+        uri.query = [uri.query, "public=#{options[:public]}"].compact.join('&') unless options[:public].nil?
```

Comment:
We want not to add `public` parameter to the request if `public` has a value that is not `true`.

Reply:
And please add tests about `public` handling.